### ISSUE TITLE
Error handling failed when Bitcoin core has an unencrypted wallet, but wallet password is passed

### ISSUE THREAD

**Issue by ripcurlx** [bisq-network/bisq2#115]
When trying to connect to Bitcoin core passing a passphrase and the wallet is unencrypted it fails with
 
```
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Throwable.toString()" because the return value of "java.lang.Throwable.getCause()" is null
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Throwable.toString()" because the return value of "java.lang.Throwable.getCause()" is null

	at bisq.wallets.bitcoind.rpc.RpcClient.rpcAuthenticationFailed(RpcClient.java:77)
	at bisq.wallets.bitcoind.rpc.RpcClient.invoke(RpcClient.java:54)
	at bisq.wallets.bitcoind.BitcoindWalletBackend.walletPassphrase(BitcoindWalletBackend.java:141)
	at bisq.wallets.bitcoind.BitcoindWallet.initialize(BitcoindWallet.java:60)
	at bisq.wallets.WalletService.lambda$initialize$0(WalletService.java:47)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1800)
```

Because `Error: running with an unencrypted wallet, but walletpassphrase was called.` is returned and 
```
private boolean rpcAuthenticationFailed(Throwable t) {
        return t.getCause().toString().contains("401");
    }
```
doesn't handle the case properly when no cause exists.

**Comment by ripcurlx** [comment]
The weird thing is that I actually have protected my wallet in Bitcoin core with a passphrase.

**Comment by ripcurlx** [comment]
> Bitcoin Core supports multiple wallets (you can load and use many simultaneously). In Bisq 2, we create a new wallet. We don't touch any existing/loaded wallets.

Ah - just recognized the newly created wallet. 👍

**Comment by alvasw** [comment]
Fixed in #121
